.vscode/extensions.json

```diff
@@ -1,7 +1,8 @@
 {
   "recommendations": [
     "dbaeumer.vscode-eslint",
-    "octref.vetur",
-    "rvest.vs-code-prettier-eslint"
+    "rvest.vs-code-prettier-eslint",
+    "vue.volar",
+    "Vue.vscode-typescript-vue-plugin"
```

Comment:
With Vue 2.7+, Volar + TS Vue Plugin are the recommended extensions, Vetur doesn't have enough support to handle the new stuff.